openmmforcefields/generators/template_generators.py

```diff
@@ -236,9 +236,13 @@ def _generate_unique_atom_names(self, molecule):
             The molecule whose atom names are to be modified in-place
         """
         from collections import defaultdict
+
+        # OpenFF Toolkit v0.11.0 removed Atom.element and replced it with Atom.symbol, etc.
+        uses_old_api = hasattr(molecule.atoms[0], "element")
+
         element_counts = defaultdict(int)
         for atom in molecule.atoms:
-            symbol = atom.element.symbol
+            symbol = atom.element.symbol if uses_old_api else atom.symbol
```

Comment:
Oh, that's an unexpected change.

@mattwthompson, can you point to some information about the motivation for this? Is the idea that `Atom` objects don't have to be physical elements anymore?

Reply:
And if `symbol` supports non-physical elements or made-up symbols now, is there an API for telling whether the element is a real element or not?

Reply:
This is the PR where the change happened: https://github.com/openforcefield/openff-toolkit/pull/1182
It comes from switching to a different package.

Reply:
@jchodera this is the only modification needed to get openmmforcefields working with the changes in the openff-toolkit/openff-element and given they are done in a backwards compatible way, I am happy with them. If you are happy with it, I can make a new PR add this changes + tests the latest version of the openff-toolkit.